der/src/bytes_owned.rs

```diff
@@ -64,6 +64,29 @@ impl<'a> DecodeValue<'a> for BytesOwned {
     type Error = Error;
 
     fn decode_value<R: Reader<'a>>(reader: &mut R, header: Header) -> Result<Self> {
+        // Reassemble indefinite length string types
+        if header.length.is_indefinite() && !header.tag.is_constructed() {
+            // Parse constructed indefinite length data
```

Comment:
@tarcieri, I’ve been following your recent work related to
- https://github.com/RustCrypto/formats/issues/779

I wasn’t following this change… the comment says ‘constructed’ and the goal is constructed OctetString support, but the condition says `! is_constructed`.
I must be missing something.

Reply:
It's a bit confusing. The `Tag` type is designed for DER where the constructed encoding of e.g. string types is disallowed.

So that's essentially asking "would this type of data be encoded with the constructed bit in DER?". If it is (for example, a `SEQUENCE`), then it avoids trying to decode it.